common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -59,10 +59,12 @@
         <span id="answer_${id}"></span>
     </fieldset>
     <div class="indicator-container">
-        % if input_type == 'checkbox' or not value:
-            <span class="status ${status.classname if show_correctness != 'never' else 'unanswered'}" id="status_${id}" aria-describedby="${id}-legend" data-tooltip="${status.display_tooltip}">
-                <span class="sr">${status.display_tooltip}</span>
-            </span>
+        % if input_type == 'checkbox' or status.classname == 'unanswered':
```

Comment:
This is an update to the "not value". This will ensure there is always a status for unanswered problems and when they are answered the status will shift for radio problems, but stay in this container for check box problems.